Dockerfile

```diff
@@ -1,10 +1,14 @@
 FROM jboss/keycloak:9.0.0
 USER root
-RUN curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 > /usr/local/bin/jq \
-    && chmod +x /usr/local/bin/jq
-USER 1000
-COPY realm-export.json /tmp
-COPY import-realm.sh /tmp
+RUN microdnf update && microdnf install python3
```

Comment:
Did you look into combining the config files on Jenkins rather than on the auth server itself, like we were discussing last week? It would be nice not to have to install a whole programming language on the auth server just to run the deployment step.

I don't think it's worth changing the approach now because you've done so much work already and it's probably not a big problem, but I'd be interested to know what the reason is for doing it this way in case we want to change it in future.

Reply:
There is nothing to stop us doing like that. The jenkins node that runs the build will need python on it to run the scripts to generate the combined realm json for import. Main reason was this was where the existing changes were made.